e2e/tests/upgrades/genesis_test.go

```diff
@@ -37,27 +37,36 @@ type GenesisTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+// TODO: this configuration was originally being applied to `GetChains` in the test body, but it is not
+// actually being propagated correctly. If we want to apply the configuration, we can uncomment this code
+// however the test actually fails when this is done.
+// func (s *GenesisTestSuite) SetupSuite() {
+//	configFileOverrides := make(map[string]any)
+//	appTomlOverrides := make(test.Toml)
+//
+//	appTomlOverrides["halt-height"] = haltHeight
+//	configFileOverrides["config/app.toml"] = appTomlOverrides
+//
+//	s.SetupChains(context.TODO(), nil, func(options *testsuite.ChainOptions) {
+//		// create chains with specified chain configuration options
+//		options.ChainSpecs[0].ConfigFileOverrides = configFileOverrides
+//	})
+// }
+
```

Comment:
This was nasty. Previously, the configuration you see here was not actually applied, and if it is applied the test breaks (chain halts before the ica account was registered)

I will create an issue to track this, as the failure is not related to the current issue.

Reply:
https://github.com/cosmos/ibc-go/issues/6750

Reply:
https://github.com/cosmos/ibc-go/issues/6750

Reply:
I might be solving this now in the upgrade tests for v9 because I hit a similar issue when doing a bit more setup before the upgrade test. The hardcoding of haltHeight is being removed. Will just assign myself the 6750 so I can verify it later.

Reply:
Amazing thank you!